src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -370,7 +370,13 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
       // Enable labels when doing a relayout, layouts can be told to take into account the labels to avoid
       // overlap, but we need to have them enabled (nodeDimensionsIncludeLabels: true)
       this.turnNodeLabelsTo(cy, true);
-      cy.layout(LayoutDictionary.getLayout(this.props.graphLayout)).run();
+      const layoutOptions = LayoutDictionary.getLayout(this.props.graphLayout);
+      if (cy.nodes('$node > node').length > 0) {
+        // if there is any parent node, run the group-compound-layout
```

Comment:
OK, this general test is good.  One trivial comment: I think this comment would be more effective above the 'if' (no need to change).